requirements.txt

```diff
@@ -7,11 +7,11 @@ aiohttp~=3.8
 # 8.1.0+ breaks dask/distributed versions older than 2022.04.0, see here - https://github.com/dask/distributed/pull/6018
 click~=8.0.0
 # fastapi 0.78.0 requires starlette 0.19.1 which requires typing_extensions>=3.10.0
-# kfp 1.8.1 requires typing-extensions>=3.7.4,<4
+# kfp 1.8.13 requires typing-extensions>=3.7.4,<5
```

Comment:
The comment was there to explain that the limit on `typing-extensions` was due to kfp, but now it doesn't really make sense anymore. Better to just delete this comment.

Reply:
removed